src/main/blackbox/blackbox.c

```diff
@@ -1745,6 +1750,7 @@ void blackboxLogEvent(FlightLogEvent event, flightLogEventData_t *data)
         blackboxWriteUnsignedVB(data->syncBeep.time);
         break;
     case FLIGHT_LOG_EVENT_FLIGHTMODE: // New flightmode flags write
+    case FLIGHT_LOG_EVENT_ACTUAL_FLIGHTMODE: // New RC BOX flightmode flags write
```

Comment:
Think this is wrong approach. We should use FLIGHT_LOG_EVENT_FLIGHTMODE for the flight mode.
Perhaps a FLIGHT_LOG_EVENT_MSPBOX.

Reply:
FLIGHT_LOG_EVENT_FLIGHTMODE shows change of RC state only, the real flight modes can have other values.
For example - picture above
FLIGHT_LOG_EVENT_FLIGHTMODE  shows HORIZON and ANGLE on while ACC was off.
I suggest to use  FLIGHT_LOG_EVENT_ACTUAL_FLIGHTMODE instead off FLIGHT_LOG_EVENT_FLIGHTMODE   by drawing events marker.

Reply:
Yes obvious current implementation is not correctly representing flight modes but suggest fixing the current event.
We need to have a closer look about Flight Modes and Box values being used and logged.

Reply:
Yes, of course, the mode research really need. For example - the autolaunch and acrotrainer flight mides has rc boxes flags only